includes/apple-push-api/request/class-request.php

```diff
@@ -265,6 +265,10 @@ private function parse_response( $response, $json = true, $type = 'post', $meta
 				$message = __( 'Unable to fetch information for the specified article. Was it deleted?', 'apple-news' );
 			}
 
+			if ( 'DUPLICATE_ARTICLE_FOUND' === $error->code ) {
+				$message .= '.<br>Original UUID: ' . $error->value;
```

Comment:
Best to run `$error->value` through sanitize_text_field since it should just be alphanumeric. I know we've got other escaping/sanitization later but best to be safe. We also need translation on the string:
```suggestion
				// translators: UUID of original article
				$message .= '.<br>' . sprintf( __( 'Original UUID: %s', 'apple-news' ), sanitize_text_field( $error->value ) );
```